commit 39ffec281af577c766c619d568f7a7051d108b00
Author: Danny Al-Gaaf <danny.al-gaaf@bisect.de>
Date:   Wed Apr 25 09:30:56 2018 +0200

    misc: mark constructors as explicit
    
    Set 218 constructors as explicit to avoid implicit usage.
    
    Fix for cppcheck warning:
     Class has a constructor with 1 argument that is
     not explicit. Such constructors should in general be explicit for
     type safety reasons. Using the explicit keyword in the constructor
     means some mistakes when using the class can be avoided.
    
     For more information check:
      https://www.codeproject.com/Articles/28663/Explicit-Constructor-in-C
    
    Signed-off-by: Danny Al-Gaaf <danny.al-gaaf@bisect.de>

diff --git a/src/mgr/PyModuleRegistry.h b/src/mgr/PyModuleRegistry.h
index 5fb0148e710..97eff9d4c00 100644
--- a/src/mgr/PyModuleRegistry.h
+++ b/src/mgr/PyModuleRegistry.h
@@ -76,7 +76,7 @@ public:
     return modules_out;
   }
 
-  PyModuleRegistry(LogChannelRef clog_)
+  explicit PyModuleRegistry(LogChannelRef clog_)
     : clog(clog_)
   {}
 
